src/GrpcServer/CartaGrpcService.cc

```diff
@@ -88,3 +89,41 @@ grpc::Status CartaGrpcService::CallAction(
 
     return status;
 }
+
+GrpcManager::GrpcManager() : _selected_port(-1) {}
```

Comment:
is this the "dummy" constructor for tests?

Reply:
It's the dummy constructor for the default case of running the backend with gRPC disabled.